### PR TITLE
Update NAE2 to 1.6.4

### PR DESCRIPTION
**Pull request by GavinFF-SS**
Changes I think will make the pack better.

### CODE REVIEW

**Comment by IntegerLimit** [comment]
256k Recipes already exist... see https://github.com/Nomi-CEu/Nomi-CEu/blob/main/overrides/groovy/postInit/Post-Initial/Main/Mod-Specific/nae2.groovy.

Once that script is deleted, apart from that, looks good!

**Comment by GavinFF-SS** [comment]
yeah i probably shouldn't have merged nomi ceu main branch into my fork when this pull request isn't merged, mistake on my part, hopefully it doesn't mess up too much (if anything at all), sorry about that.

File: overrides/scripts/NAE2.zs
Comment (IntegerLimit) [inline overrides/scripts/NAE2.zs:1]: One change: Please delete this script file (as per my comment above about it being implemented in groovy).

Otherwise, this PR looks good!

**Review by IntegerLimit — CHANGES_REQUESTED** [review]

File: overrides/scripts/NAE2.zs
Comment (GavinFF-SS) [reply overrides/scripts/NAE2.zs:1]: alright, done

**Review by IntegerLimit — APPROVED** [review]
Just a final check: Does NAE2 1.6.4 work with our current version of AE2 (.56.4)? Or does it require .56.5?

**Comment by GavinFF-SS** [comment]
It uses AE2UEL .56.5 if I remember correctly.

**Comment by IntegerLimit** [comment]
Alright, then I'll merge this after I merge https://github.com/Nomi-CEu/Nomi-CEu/pull/793.

**Pull request merged**
